### ISSUE TITLE
Implement comments from Steinmetz

### ISSUE THREAD

**Issue by andycasey** [AnnieJumpCannon/RAVE#23]
As per email September 6 2016.

**Comment by andycasey** [comment]
Items:
- [x] 1) nomenclature: isn’t age also a physical parameter? better : atmospheric parameter?
- [x] 2) I am not sure whether TGAS will have all Tycho targets (if I am informed correctly it will not). so the number will probably be somewhat smaller. Maybe we can leave it and fix it post refereeing?
- [x] ~~3) nomenclature: does astrometry include RV (usually not) - add RV explicity, or instead: „measure position, distances, proper motions and radial velocities“~~ **[I see astrometry as the measurement of the positions and movements of stars (so can include RVs)]**
- [x] 4) Boeche et al 2013a should be added, in particular since it made a first version of fig 14.
- [x] 5) see comment 1, the numbers may change depending what exactly is in TGAS
- [x] 6) actually the numbers are higher. the numbers you are quoting are the spectra we have in DR5, which is not quite the end of the story yet (we have a number of fields with reasonable data, but with peculiarities which needs some further scrutiny, these are not in DR5)
- [x] 7) I would turn the logic slightly around, it was mainly because the wavelength reason was used for Gaia RVS which resulted in it being well studied (and this is why we used it for RAVE), and therefore it was also used for Gaia ESO. So I would put the order Gaia RVS (including the Munari 2005 paper), then RAVE, the Gaia ESO.
- [x] 8) „the teams enormous success“ reads weired when most of the team is coauthor … could you rephrase?
- [x] 9) the chemical pipeline paper is missing (Boeche et al 2011)
- [x] 10) physics based model -> physics based model like those employed in RAVE DR2-5
- [x] 11a) ignore
- [x] ~~11) This is probably the most critical (in terms of not really fulfilled) assumption since many RAVE fields had considerably fringing issues that needed to be corrected. Maybe Tomaz can add a comment?~~ **[going to hold off on this until TZ or KCF can comment further on the extent of the fringing -- and this is mentioned in discussion too]**
- [x] ~~12) maybe worth a comment that the overlap of RAVE and APOGEE is likely to increase considerably with the advent of APOGEE-south.~~ For Galah rather than not precise enough, there are not really available yet (these are all pre-releases) (would also be more friendly) **[not so much the overlap in APOGEE that killed us, it was just that the main-sequence was bad. Removed comment about GALAH, since the data aren't out yet we don't even need to consider it]**
- [x] 13) check: you make no use of any of the RAVE flags for the cross-sample selection? **[we didn't use any RAVE flags, no]**
- [ ] 14, 15)
- [x] ~~16) Wouldn’t be much better to have all figures in sync with DR5 data, in particular since they appear in parallel and refer to each other. This would appear to be much more thorough and logical.~~ **[if possible I will try and include both comparisons]**
- [x] 17) and as seen in other surveys (see APOGEE DR11)
- [x] ~~17a) Have you done a comparison with the individual elemental abundances from DR4/5, or at least when the CDR abundances are combined to an alpha? From figure 14 I would expect it to compare reasonably well (though with substantial scatter)~~ **[I have, and they agree but it is just a big ball of fluff -- it is not so informative I think to warrant an additional figure in the paper. Happy to hear opposing arguments]**
- [x] 18) at the moment I would argue for „on request“
- [x] 19) maybe specific „stellar parameters“ mean Cannon parameters here
- [x] 20) how about the globular clusters in Kunder et al (2014) with RAVE?
- [x] 21) this is already stated in DR1 and subsequent papers. Effects include varying resolution (owing to the limited optical quality of the spectrograph camera), temperature variations, fibre to fibre variation etc etc
- [x] 22) electronic access: please add that the data will also incorporated in the DR5 data base (Kunder et al).

[unrave.pdf](https://github.com/AnnieJumpCannon/RAVE/files/457082/unrave.3.pdf)

**Comment by keithhawkins** [comment]
how about the globular clusters in Kunder et al (2014) with RAVE -- We can add one cluster from that paper NGC1857 with ~13 possible members. The other clusters are either already added or only have 1-2 members.

**Comment by andycasey** [comment]
OK. Add it to the figure, but forget the other clusters.

**Comment by andycasey** [comment]
Or rather: update the `RAVEGC_DR4.fits` file to include those members, and I will update the plotting script to be a little bit more,.. beautiful 🍻

**Comment by keithhawkins** [comment]
done adding Kunder et al 2014 cluster (RAVEGC_DR4.fits has been added to my ges disk space)
